spec.html

```diff
@@ -7884,9 +8060,12 @@ <h1>Runtime Semantics: NamedEvaluation</h1>
       <emu-alg>
         1. Return InstantiateAsyncArrowFunctionExpression of |AsyncArrowFunction| with argument _name_.
       </emu-alg>
-      <emu-grammar>ClassExpression : `class` ClassTail</emu-grammar>
+      <emu-grammar>ClassExpression : DecoratorList? `class` ClassTail</emu-grammar>
       <emu-alg>
-        1. Let _value_ be the result of ClassDefinitionEvaluation of |ClassTail| with arguments *undefined* and _name_.
+        1. If |DecoratorList_opt| is present, then
```

Comment:
```suggestion
        1. If |DecoratorList| is present, then
```

here and in other similar places with `|DecoratorList_opt| is present`.

Reference for consistency: https://tc39.es/ecma262/#sec-static-semantics-lexicallydeclarednames

Reply:
There are [some](https://tc39.es/ecma262/#sec-runtime-semantics-propertydestructuringassignmentevaluation) [few](https://tc39.es/ecma262/#sec-runtime-semantics-classfielddefinitionevaluation) [parts](https://tc39.es/ecma262/#sec-runtime-semantics-classdefinitionevaluation) where `_opt` is preserved, but it feels like an error.